std_msgs/Header head
int32 time
int8 bat_perc
float32 bat_level

float32 temperature
float32 pressure
float32 RH

float32 O2_conc
float32 H2S_conc
float32 CO_conc
float32 CH4_conc

bool O2_alarm
bool H2S_alarm
bool CO_alarm
bool CH4_alarm

geometry_msgs/Vector3 accel